### ISSUE TITLE
No Issue but an Info: macOS Wrapper 

### ISSUE THREAD

**Issue by MacThings** [MaurycyLiebner/eZeus#6]
I created an macOS Wrapper. Simply put the contents of the DATA Folder to the DATA Folder of the wrapper. Done.

https://github.com/MacThings/eZeus-wrapper

![grafik](https://github.com/user-attachments/assets/8de7493e-e705-428b-b313-69cf63f6e328)

**Comment by MacThings** [comment]
Got it natively running ... must lok for SDL2 Problem now:

<img width="1262" alt="grafik" src="https://github.com/user-attachments/assets/ed66ba89-ec0b-4685-a493-e2d727ab410e" />

**Comment by MacThings** [comment]
Managed it to compile it under macOS. It's running natively now. :D

<img width="1306" alt="grafik" src="https://github.com/user-attachments/assets/de082a41-4fb5-45a8-bd33-f1055575462b" />

**Comment by MacThings** [comment]
Eine angepasste Text Datei für Deutsch liegt ebenfalls in der APP dabei.

![grafik](https://github.com/user-attachments/assets/86918fc0-7aca-4605-b8c4-11872a371ab5)

**Comment by Chimera16** [comment]
Nice work!

**Comment by MacThings** [comment]
I hope development is not stopped. Since 4 Months no update. :(

**Comment by MaurycyLiebner** [comment]
I started university (biomedical engineering), so currently I don't have time to continue the project. I have some ideas for future development, but I will not be able to work on them until summer vacation, unless, of course,  I fail my exams this semester. ;)

**Comment by Chimera16** [comment]
I wish you the best for your exams! I hope you will find sooner or later some time to finish this project, as it has a lot of potential, and of course, that you will not fail your exam! :)

**Comment by MacThings** [comment]
Yes, a lot of potential. (: I have running the original Zeus (with HD Patch) with CrossOver here on my Mac but a native Version (with qol features) would be a very good thing.

**Comment by MacThings** [comment]
I added 0.8.1-Beta.8 macOS Binaries here:

https://github.com/MacThings/eZeus/releases/tag/Zeus-Binaries
